lib/wallet.js

```diff
@@ -315,6 +315,74 @@ class WalletClient extends Client {
     return this.get(`/wallet/${id}/coin/${hash}/${index}`);
   }
 
+  /**
+   * Get NameState from wallet
```

Comment:
How about we make this comment more informative? Perhaps, "Get name info for the given name."